internal/fsutil/create_test.go

```diff
@@ -122,14 +192,23 @@ func (s *S) TestCreate(c *C) {
 
 		c.Assert(err, IsNil)
 		c.Assert(testutil.TreeDump(dir), DeepEquals, test.result)
+
 		// [fsutil.Create] does not return information about parent directories
 		// created implicitly. We only check for the requested path.
-		entry.Path = strings.TrimPrefix(entry.Path, dir)
-		// Add the slashes that TreeDump adds to the path.
-		slashPath := "/" + test.options.Path
-		if test.options.Mode.IsDir() {
-			slashPath = slashPath + "/"
+		if entry.Link != "" && entry.Mode&fs.ModeSymlink == 0 {
```

Comment:
I see you removed the isHardLink variable, if that is the case can you add a comment like:
```go
if entry.Link != "" && entry.Mode&fs.ModeSymlink == 0 {
    // entry is a hardlink.
    ....
```

Reply:
Fixed in [3e8d872](https://github.com/letFunny/chisel/pull/17/commits/3e8d872c21ef3ed81e75c5319a052f872feb8b25).